Sources/Attachments.php

```diff
@@ -199,7 +199,6 @@ protected function processAttachments()
 			list ($context['attachments']['quantity'], $context['attachments']['total_size']) = $smcFunc['db_fetch_row']($request);
 			$smcFunc['db_free_result']($request);
 		}
-
```

Comment:
Like @MissAllSunday, I'm not a fan of removing all the vertical whitespace between if/elseif/else blocks. Technically it is unnecessary, sure, but sometimes it really helps legibility.